src/sections/wallet/assets/table/actions/WalletAssetsTableActionsMob.tsx

```diff
@@ -91,11 +93,35 @@ export const WalletAssetsTableActionsMob = ({
           <div sx={{ flex: "column", gap: 12 }}>
             <div>
               <div sx={{ flex: "row", gap: 12 }}>
-                <Button sx={{ width: "100%" }} size="small">
+                <Button
```

Comment:
Nemáme nějakou `<ButtonLink>` componentu ? je to lepší z hlediska accessibility

Reply:
Takove komponenty nepouzivame

Reply:
Mohl by jsi ji prosím udělat ? Mělo by stačit wrappnout `<Button>` do `<a />`

Reply:
Něco podobného už existuje v `<TabLink />` 

src/components/Tabs/TabLink.tsx